source/administration/command-line-tools.rst

```diff
@@ -586,6 +587,27 @@ mattermost config migrate
     .. code-block:: none
 
        ./mattermost config migrate  path/to/config.json "postgres://mmuser:mostest@dockerhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"
+       
+mattermost config reset
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Resets the value of a config setting by its name in dot notation or a setting section to the default value. Accepts multiple values for array settings. When no paramters are given, it will reset all config settings. 
```

Comment:
```suggestion
    Resets the value of a config setting by its name in dot notation or a setting section to the default value. Accepts multiple values for array settings. When no parameters are given, it will reset all config settings. 
```